design-doc.md

```diff
@@ -0,0 +1,62 @@
+# 10bis project
+
+## Motivation
+
+An single-page application that simplifies the 10bis food ordering process by allowing users to choose from their top 3 most-ordered meals. Because in the end, we are creatures of habit and usually order the same meal everyday, so why not limit the effort it takes to order?
+
+## Scope
+
+### Flow
+
+1. User login attempt
+
```

Comment:
I recommend using Use-case diagrams (you can read about it in "Righting-Software" P.56)